tgui/docs/archives/the-holy-bible-of-tgui-by-arcane.md

```diff
@@ -0,0 +1,210 @@
+# The Holy Bible of TGUI with Arcane and a shot of Vodka
```

Comment:
This file looks out of date?

Reply:
It is, I added it for fun.

Reply:
Add some kind of clarification that this is old and refers to a deprecated version.